### PR TITLE
onEndReached mistyped

### PR DESCRIPTION
**Pull request by maarekj**
Fix type of onEndReached
(see issue #149)

### CODE REVIEW

File: src/components/flatList.rei [hunk 1]
```diff
@@ -48,12 +58,19 @@ let make:
...
+                      "offset": int,
+                      "index": int
+                    }
+                      =?,
     ~horizontal: bool=?,
     ~initialNumToRender: int=?,
     ~initialScrollIndex: int=?,
     ~numColumns: 'int=?,
-    ~onEndReached: {. "distanceFromEnd": float}=?,
+    ~onEndReached: {. "distanceFromEnd": float} => unit=?,
```
Comment (maarekj) [inline src/components/flatList.rei]: that's the essence of the pull request.

File: src/components/sectionList.re [hunk 2]
```diff
@@ -147,7 +134,7 @@ let make:
     ~sectionSeparatorComponent: separatorComponent('item)=?,
     ~extraData: 'extraData=?,
     ~initialNumToRender: int=?,
-    ~onEndReached: {. "distanceFromEnd": float}=?,
+    ~onEndReached: {. "distanceFromEnd": float} => unit=?,
```
Comment (maarekj) [inline src/components/sectionList.re]: that's the essence of the pull request.

File: src/components/sectionList.rei [hunk 3]
```diff
@@ -69,7 +76,7 @@ let make:
     ~sectionSeparatorComponent: separatorComponent('item)=?,
     ~extraData: 'extraData=?,
     ~initialNumToRender: int=?,
-    ~onEndReached: {. "distanceFromEnd": float}=?,
+    ~onEndReached: {. "distanceFromEnd": float} => unit=?,
```
Comment (maarekj) [inline src/components/sectionList.rei]: that's the essence of the pull request.

**Comment by zploskey** [comment]
Looks like it also needs to be changed in flatList.re.

**Pull request closed without merging**